superdesk/text_checkers/ai/__init__.py

```diff
@@ -156,6 +158,70 @@ def create(self, docs, **kwargs):
         return [0]
 
 
+class AIImageResource(Resource):
+    schema = {
+        "service": {
+            "type": "string",
+            "required": True,
+        },
+        "item": {
```

Comment:
would just call this `items` if it's a list and not a single one

Reply:
or make it `item` which won't be a list but a dictionary like in the other services

Reply:
Thank you for the feedback. We have now adjusted this and refactored from "item" to "items".